docs/plugins.md

```diff
@@ -1,87 +1,77 @@
 # Plugins
 
-BitOps' default image is actually BitOps core with 4 plugins pre-installed:
-* [bitops-terraform-plugin](https://github.com/bitovi/bitops-terraform-plugin.git)
-* [bitops-ansible-plugin](https://github.com/bitovi/bitops-ansible-plugin.git)
-* [bitops-cloudformation-plugin](https://github.com/bitovi/bitops-cloudformation-plugin.git)
-* [bitops-helm-plugin](https://github.com/bitovi/bitops-helm-plugin.git)
+BitOps' default image contains BitOps base along with pre-installed plugins:
```

Comment:
`default` could be ambiguous. either call it `latest` or be specific and call out `omnibus`